gui/default/syncthing/core/syncthingController.js

```diff
@@ -2538,13 +2553,11 @@ angular.module('syncthing.core')
             $scope.config.folders = folderList($scope.folders);
             recalcLocalStateTotal();
 
-            $scope.saveConfig();
+            $scope.saveConfig().then(function () {
+                hideModal('#editFolder');
+            });
         };
 
-        function hideFolderModal() {
-            $('#editFolder').modal('hide');
-        }
```

Comment:
I have removed this completely as no longer needed (and replaced by the `showModal` and `hideModal` functions.